web/pandas/community/ecosystem.md

```diff
@@ -345,11 +345,12 @@ which pandas excels.
 
 ## IO
 
-### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas)*: A semantic, compact and reversible JSON-pandas converter*
+### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas): *A semantic, compact and reversible JSON-pandas converter*
 
-NTV-pandas provides a JSON converter with more data types than the ones supported by pandas directly.
+`NTV-pandas` provides a JSON converter with more data types than the ones supported by pandas directly.
```

Comment:
We use backticks for code. If this was the name of the module it could be ok, but better revert this, as it's misleading.